app/components/ui/Navigation/Navigation.tsx

```diff
@@ -78,7 +88,19 @@ export const Navigation = () => {
   };
 
   return (
-    <>
+    <InstantSearch
+      searchClient={searchClient}
+      indexName={`${config.ALGOLIA_INDEX_PREFIX}_services_search`}
+      routing={{
+        router: history({
+          windowTitle({ production_services_search: { query } }) {
+            const queryTitle = query ? `Results for "${query}"` : "Search";
+
+            return queryTitle;
+          },
+        }),
```

Comment:
This replaces code from https://github.com/Exygy/askdarcel-web/pull/201/files#diff-dc73b48e044f976629cb7d8ecbf048e44735cb6df01480058b5046407ed3795cL111